orchestrator-salt/src/main/resources/salt/salt/cloudera/manager/start.sls

```diff
@@ -12,6 +12,7 @@ start_server:
   service.running:
     - enable: True
     - name: cloudera-scm-server
+{# Protect this via a grains[prewarmed] check}
```

Comment:
Even the base image has the IPA packages installed. We could remove this, but need to validate our oldest available images in prod whether they have IPA installed as well.